usage of visual variables and other renderer options using ent profile

CMPeng opened the pull request:
https://github.com/ArcGIS/geosaurus/issues/1854

1. using lower case for ipynb file name
2. using ent profile
3. removing extra spacing and newlines in the ipynb

rohitgeo commented on guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb:
```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```
<p>@CMPeng I'm not sure if I'm missing something, but it seems that we're only using the id from the `map6_layers`. Do we need such a big dict with popup info, field info, etc?</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

AtmaMani replied:
<p>Can you rewrite the sentence "To begin with ... numeric values in your data".</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

AtmaMani replied:
<p>Can we create a FeatureLayer object that points to this URL and add it in this snippet and allow other snippets that use this layer as</p><p><br></p><p>```</p><p>usa_flayer = FeatureLayer(url="...")</p><p>`map1.add_layer(usa_flayer, options={"renderer":"...","field_name":"...")</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

AtmaMani replied:
<p>Here we need to distinguish between `autocast` and smart mapping. We can say that "While smart mapping will automatically choose the class breaks, users can use autocasting if they want finer control by providing their own class breaks and symbology for each class".</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

AtmaMani replied:
<p>Can you rewrite "Besides taking advantage... role in mapping"</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

AtmaMani replied:
<p>Text here pertains to map viewer UI. Can you either rewrite or drop this paragraph?</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

AtmaMani replied:
<p>Code in section 2.1 does not contain any specific instructions to use unique values for the railway lines, but the railway lines are in different color. How does this work?</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

CMPeng commented:
Hi @rohitgeo 
Yes the dictionary contains pop-up info that can be removed (since it is just to control the format and what fields to be displayed when click-and-show the pop-ups).
Please see the new commit for the latest version with the pop-up removes. Thanks!

AtmaMani commented:
@CMPeng this is a great guide. In addition to the specific points, can rename the notebook to "advanced-cartography-part1" and "advanced-cartography-part2"? You can put sections 2,3 into the part2

CMPeng commented:
Thanks @AtmaMani for the detailed suggestions, and I have addressed them in the new commit.
Two points to emphasize -
1. "Code in section 2.1 does not contain any specific instructions to use unique values for the railway lines, but the railway lines are in different color. How does this work?" (Second to the last comment you made) 
- because the MapServer itself contains the render info, so this railway example can draw different colors of polylines without customization. I have put more text descriptions there, and guide users to use the default renderer, or customize their own style in the sample underneath this one.
2. "In addition to the specific points, can rename the notebook to "advanced-cartography-part1" and "advanced-cartography-part2"? You can put sections 2,3 into the part2"
- addressed. Also added subtle linkage sentences at the end of part 1, and in the beginning of part 2.
